cmake/gflags.cmake

```diff
@@ -18,19 +18,18 @@ if("${gRPC_GFLAGS_PROVIDER}" STREQUAL "module")
   endif()
   if(EXISTS "${GFLAGS_ROOT_DIR}/CMakeLists.txt")
     add_subdirectory(${GFLAGS_ROOT_DIR} third_party/gflags)
-    if(TARGET gflags_static)
-      set(_gRPC_GFLAGS_LIBRARIES gflags_static)
-      set(_gRPC_GFLAGS_INCLUDE_DIR "${CMAKE_CURRENT_BINARY_DIR}/third_party/gflags/include")
-    endif()
+    set(_gRPC_GFLAGS_LIBRARIES gflags::gflags)
```

Comment:
why change from gflags_static to just  gflags and getting rid of the if?

Reply:
because the changes in gflags CMakeLists defaulted to static, see https://github.com/gflags/gflags/blob/master/CMakeLists.txt#L496